src/emailservice/Dockerfile

```diff
@@ -8,15 +8,15 @@ FROM base as builder
 
 WORKDIR /tmp
 
-COPY Gemfile Gemfile.lock .
+COPY ./src/emailservice/Gemfile ./src/emailservice/Gemfile.lock ./
```

Comment:
Use `Dockerfile` from root's demo directory instead (like the other services approach)